ui/packages/consul-ui/app/instance-initializers/container.js

```diff
@@ -13,7 +13,7 @@ export default {
     let repositories = container
       .get('container-debug-adapter:main')
       .catalogEntriesByType('service')
-      .filter(item => item.startsWith('repository/'));
+      .filter(item => item.startsWith('repository/') || item === 'ui-config');
```

Comment:
`ui-config` should be a repository. Added a little workaround here, rather than moving things around, but that should be changed as soon as possible to make it work like the rest of the application (including a completely async API)